channels/in.m3u

```diff
@@ -379,9 +381,13 @@ http://103.199.160.85/Content/puthuyugam/Live/Channel(Puthuyugam)/index.m3u8
 #EXTINF:-1 tvg-id="RPlusNews.in" tvg-name="R Plus News" tvg-country="IN" tvg-language="" tvg-logo="" group-title="News",R Plus News (360p)
 https://cdn.smartstream.video/smartstream-us/rplus/rplus/playlist.m3u8
 #EXTINF:-1 tvg-id="RajDigitalPlus.in" tvg-name="Raj Digital Plus" tvg-country="IN" tvg-language="Hindi" tvg-logo="" group-title="Movies",Raj Digital Plus (404p)
-http://acv.asianetmobiletvplus.com/channels/rajdigital_htfuqchouekpefzxodtbzaojtryxpqkx_hls/rajdigital_htfuqchouekpefzxodtbzaojtryxpqkx_master.m3u8
+http://198.144.149.82:8080/NOTV/RELAXTV/index.m3u8?token=GTRajdigital_htfuqchouekpefzxodtbzaojtryxpqkx_master.m3u8
```

Comment:
Still not working.
![image](https://user-images.githubusercontent.com/7253922/130039247-25691fe5-0dd7-45ee-ad21-ad0614c0bc0b.png)